styles/colors-carbon.less

```diff
@@ -64,6 +64,10 @@
 @agate-colorpicker-bg-color: rgba(0, 0, 0, 0.5);
 @agate-colorpicker-palette-shadow: @agate-popup-shadow;
 
+// DateTimePicker
+// ---------------------------------------
+@agate-datetimepicker-active-container-bg-color: @agate-accent;
```

Comment:
These can be optimized:  Set @agate-datetimepicker-active-container-bg-color: @agate-accent; in colors-base and remove the duplicate declarations from colors-[skin].less.
For electro skin, you can set it to transparent or inherit